### DISCUSSION TITLE
Is there any timeline available for v2 to be promoted out of beta?

### DISCUSSION THREAD

**Discussion by rathpc** [ueberdosis/tiptap#3595] [General Questions]
Hello! I was just curious if there is any sort of timeline for when v2 will be out of beta? Thanks in advance!

**Comment by hemju** [comment]
Hi, it would be great to get some information on the current state of the project. The project was active last year, but till the end of the year it became stale. That is not necessarily a bad thing, it also could mean that the API got stable. However, since there is no final v2, many open issues and open pull requests, I wondering about the plans for this project.
